README.md

```diff
@@ -1,2 +1,14 @@
 # ops-requests
-Requests about infrastructure operations
+
+Use this repo's issue tracker to direct requests to the infrastructure team (mainly @lgierth).
+
+Make sure that:
+
+- the request is specific
+- the request is actionable
+- the request is prioritizable
+  - e.g. has a specific timeframe
```

Comment:
Updated everything a bit -- I think it might actually be wiser to go without issue labels for now